README.md

```diff
@@ -51,9 +51,9 @@ Visit our [read the docs page](https://edapython.readthedocs.io/en/latest/source
 This is a working example:
 
 ```
-import altair as alt
```

Comment:
Should we not keep altair in as an import?

Reply:
No, we don't need it. Our package will import it.

Reply:
But the helper function needs to use numpy and pandas, otherwise, we don't need to import these two packages.